src/main/java/com/enterprise/redcord/service/UserServiceStub.java

```diff
@@ -0,0 +1,19 @@
+package com.enterprise.redcord.service;
+
+import com.enterprise.redcord.dto.User;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserServiceStub implements IUserService{
+    @Override
+    public User saveUser(User user) throws Exception {
+        return null;
+    }
+
+    @Override
+    public List<User> fetchAllUsers() {
+        return null;
+    }
```

Comment:
This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality .
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.